app/public/js/localSprites.js

```diff
@@ -151,7 +183,8 @@ function removeFinishedLocalSprites() {
 var localSprites = [];
 var twilightTilesImage = requireImage('/gfx/jetrel/twilight-tiles.png'),
 fireballBImage = requireImage('/gfx/fireball/fireballB.png'),
-fireballContrailAImage = requireImage('/gfx/fireball/fireballContrailA.png');
+fireballContrailAImage = requireImage('/gfx/fireball/fireballContrailA.png'),
+customBlocksAImage = requireImage('gfx/customBlocksA.png');
```

Comment:
There is a bit of an issue with the image paths right now where using `gfx/...` instead of `/gfx/...` will break the page if the user navigates to a specific zone. I'm going to fix this on master when I merge in, but just keep it in mind. I probably need to find a better solution to this at some point because the current solution doesn't let us host the game at a subpath, but for now using `/gfx/` is good enough.